dbus-cxx-uv/unit-tests/uv-caller.cpp

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later OR BSD-3-Clause
+// SPDX-License-Identifier: LGPL-3.0-or-later OR BSD-3-Clause
+
+#include <uv.h>
+
+#include <dbus-cxx-uv.h>
+
+#include "calleeclass.h"
+
+int value = 0;
+
+static void timer_cb(uv_timer_t* h)
+{
+    auto methodref = *reinterpret_cast<std::shared_ptr<DBus::MethodProxy<int(int,int)>>*>(h->data);
+    value = (*methodref)( 5, 10 ); 
+    uv_stop(uv_default_loop());
+}
+
+int main(int argc, char** argv){
+    std::shared_ptr<DBus::Dispatcher> disp;
+    bool use_dispatcher = true;
+    std::string dispatch_type( argv[ 1 ] );
+
+    if( dispatch_type == "standalone" ){
+        //use_uv_dispatcher = false;
+        fprintf(stderr, "libuv doesn't support threaded/standalone");
+        return -1;
+    }
+
+    if( use_dispatcher ){
```

Comment:
is there some reason to change the dispatcher used?  If the test is for libuv, the test should just use the libuv dispatcher.

Reply:
There's none really. I was mimicking the Qt integration which has a threaded dispatcher option but I haven't any plans to add one for libuv. Removed.